pyproject.toml

```diff
@@ -69,6 +69,8 @@ dataset = "^1.5.2"
 psutil = "^7.0.0"
 weasyprint = "^62.0"
 pdfminer-six = "^20250506"
+authlib = "^1.6.0"
+oauth2-provider = "^0.0"
```

Comment:
Are you sure you're using the right package?

- https://pypi.org/project/oauth2_provider/

Didn't you mean `django-oauth2-provider` or something else?

`ModuleNotFoundError: No module named 'oauth2_provider.contrib'`

<img width="1092" height="698" alt="image" src="https://github.com/user-attachments/assets/302107a4-dffd-4292-a92d-fb6db41731d2" />